.travis/release.sh

```diff
@@ -9,7 +9,7 @@ fi
 BASE_BUILD_IMAGE=${BASE_BUILD_IMAGE:-"jaegertracing/jaeger-operator"}
 OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
 OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d\.]+)")
-JAEGER_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)")
+JAEGER_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+\.[\d]+)" | head -n 1)
```

Comment:
Why did you add the `| head -n 1`? Doesn't seem to be required.

Reply:
When we have four parts, the regex matches two digits plus two digits, generating two matches. The `head -n 1` part will get only the first one.

```
$ echo $OPERATOR_VERSION
1.13.1.2
$ echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)"
1.13
1.2
$ echo ${OPERATOR_VERSION} | grep -Po "([\d]+\.[\d]+)" | head -n 1
1.13
```

The same would happen if we have six parts in the operator version. Very unlikely, so, `head -n` is pretty much unnecessary, but it's not hurting either.

Reply:
But the regex matches 3 digits now - it doesn't have any impact, so can be left, but just wasn't sure why had been added.